common/agentinstall/params.go

```diff
@@ -17,7 +17,8 @@ func NewParams(apiKey string, options ...func(*Params) error) (*Params, error) {
 	params := &Params{
 		apiKey: apiKey,
 	}
-	return common.ApplyOption(params, WithLatest(), options)
+	options = append([]func(*Params) error{WithLatest()}, options...)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Is this different from 
```
options = append(options, []func(*Params) error{WithLatest()})
```
?

Is it because you want first to apply `WithLatest` ?

Reply:
The order matters. `WithLatest` should be applied first and maybe override by an option.